mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -3,13 +3,21 @@ AUI.add(
 	function(A) {
 		var EVENT_CLICK = 'click';
 
+		var TPL_FAILURE_MESSAGE = '<div class="alert alert-error">' +
+				'{warningMessage} ' +
+			'</div>';
+
 		var TPL_FINISHED_TASK = '<li>' +
 				'{taskFinished}' +
 				'<button>' +
 					'<i class="icon-trash"></i>' +
 				'</button>' +
 			'</li>';
 
+		var TPL_SUCCESS_MESSAGE = '<div class="alert alert-success">' +
+				'{taskAdded}' +
```

Comment:
You abstracted the success message template, why then have ```{taskAdded}``` param? It should be a name relevant to abstracted template, not to implementation to a successfully added task message. For example, ```{message}```.